### PR TITLE
requests acceptation fix

### PR DESCRIPTION
**Pull request by jakubsolecki**

### CODE REVIEW

**Comment by MrDenkoV** [comment]
Wow, great work! すごい！

File: enroll/schema/request/schema.py [hunk 1]
```diff
@@ -24,14 +24,14 @@ def mutate(root, info, enrollment_id, class_time_id, comment=""):
...
 
-        try:
-            request = StudentRequest.objects.get(enrollment=enrollment)
-        except StudentRequest.DoesNotExist as e:
-            request = StudentRequest.objects.create(
-                enrollment=enrollment, comment=comment, active=True, lecturer=lecturer
-            )
+        # try:
+        #     request = StudentRequest.objects.get(enrollment=enrollment, active=True)
+        # except StudentRequest.DoesNotExist as e:
```
Comment (maciektr) [inline enroll/schema/request/schema.py]:
```suggestion
```

File: enroll/tests/accepting_test.py [hunk 2]
```diff
@@ -81,3 +81,34 @@ def test_accepts_request_to_any(self):
         )
 
         assert result == {"data": {"acceptRequest": {"accepted": True}}}
+
+    def test_accepts_different_lecturer(self):
```
Comment (maciektr) [inline enroll/tests/accepting_test.py:85]: ❤️

File: enroll/schema/request/schema.py [hunk 3]
```diff
@@ -24,14 +24,14 @@ def mutate(root, info, enrollment_id, class_time_id, comment=""):
...
+        request = StudentRequest.objects.create(
```
Comment (maciektr) [inline enroll/schema/request/schema.py:29]: What about indempotency of this mutation?

**Comment by MrDenkoV** [comment]
Weak sheep following their leader...

**Pull request merged**